spec/lambda_party_spec.rb

```diff
@@ -1,9 +1,76 @@
+require 'pry'
+require "lambda_party/request"
+
 RSpec.describe LambdaParty do
```

Comment:
With a decent spec_helper we might be able to reduce this to `describe`.  You could also consider doing a class here TestParty that includes LambdaParty to test that functionality.